configFileProductions/spring2024ConfigFiles/nunuH_Hxx/LCFIJetEnabled/whizardddsimgaudi.conf

```diff
@@ -14,15 +14,15 @@ Arguments = --enableLCFIJet
 
 [Production Parameters]
 machine = ee
-prodGroup = fcc_ee_spring2024_HiggsCouplings_LCFIJet
+prodGroup = fcc_ee_spring2024_HiggsCouplings_LCFIJet_BeamSpotDec2022
 
 softwareVersion = key4hep_241128
 generatorApplication = whizard2
 simulationApplication = ddsim
 reconstructionApplication = gaudiapp
 generatorSteeringFile = wzp6_ee_nunuH_Htautau_ecm240.sin, wzp6_ee_nunuH_Hgg_ecm240.sin, wzp6_ee_nunuH_Hdd_ecm240.sin, wzp6_ee_nunuH_Huu_ecm240.sin, wzp6_ee_nunuH_Hss_ecm240.sin, wzp6_ee_nunuH_Hcc_ecm240.sin, wzp6_ee_nunuH_Hbb_ecm240.sin
 
-configVersion = key4hep-devel-3
+configVersion = key4hep-devel-4
```

Comment:
New packages need to be registered in ilcdirac, uploading tarballs is not enough.

Reply:
Thx for registering the package.